src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1527,6 +1527,58 @@ internal bool IsRealSigned
             Predicate<ISymbol> filterOpt,
             CancellationToken cancellationToken);
 
+        internal bool StartSourceChecksumCalculation(DebugDocumentsBuilder documentsBuilder, DiagnosticBag diagnostics)
```

Comment:
>StartSourceChecksumCalculation [](start = 22, length = 30)

Generally when I see a `Start` method I expect an `End` method to be paired with it.  Not seeing that here.